### ISSUE TITLE
Does this maintain functionality?

### ISSUE THREAD

**Issue by Andavin** [bobobo1618/ninesleep#11]
Does this maintain the limited functionality in the Eight Sleep app following the modification?

Also, obviously, no guarantees from anyone who answers, but, to your knowledge, is Eight Sleep aware of the modifications or should they, theoretically, be none-the-wiser? Has anyone been able to get support after doing this?

I appreciate any information!

**Comment by bobobo1618** [comment]
No, this replaces the software Eight Sleep uses to interact with the device, so the app won't remain functional.

If you put the files back on the SD card and reset again, it should return to factory state so you can use the app again.

I have no idea whether Eight Sleep would be aware of modifications but considering it requires opening the device, I imagine they'd be able to figure it out without too much trouble if you had to send the device back to them for some reason.

If you're at all worried about support, I'd recommend against using this project. What I've done isn't a polished easy-to-use product for the average consumer; it's a hack for those who are willing and able to screw around with their device at their own risk.
